tests/unit/borg/test_create.py

```diff
@@ -646,6 +646,53 @@ def test_create_archive_with_sources_and_used_config_paths_calls_borg_with_sourc
         global_arguments=flexmock(log_json=False, used_config_paths=['/etc/borgmatic/config.yaml']),
     )
 
+def test_create_archive_with_sources_and_used_config_paths_with_store_config_files_false_calls_borg_with_sources_and_no_config_paths():
```

Comment:
Disclaimer: this is Copilot generated. I just took a glance (lines 691 and 677) and it made sense + it passed.